draft-packages/well/docs/Well.stories.tsx

```diff
@@ -1,143 +1,159 @@
 import React from "react"
-import { StoryFn } from "@storybook/react"
-import { TextField } from "@kaizen/draft-form"
-import { Well } from "@kaizen/draft-well"
-import { Heading, Paragraph } from "@kaizen/typography"
-import { StoryWrapper } from "../../../storybook/components/StoryWrapper"
+import { Meta, StoryFn, StoryObj } from "@storybook/react"
+import { Heading, Paragraph } from "../../../packages/typography"
```

Comment:
Why did this change? 👀

Reply:
Not a fan of introducing packages just for docs when we can reference them directly

Reply:
That's fair :) Although being a monorepo, I'm pretty sure the packages are all just aliases anyway. We might need to tell eslint to ignore this check for stories though.

Reply:
Which check?

Reply:
The something something dependencies one